### ISSUE TITLE
SMILES Enumeration as Data Augmentation for Neural Network Modeling of Molecules

### ISSUE THREAD

**Issue by agitter** [greenelab/deep-review#282]
https://arxiv.org/abs/1703.07076

> Simplified Molecular Input Line Entry System (SMILES) is a single line text representation of a unique molecule. One molecule can however have multiple SMILES strings, which is a reason that canonical SMILES have been defined, which ensures a one to one correspondence between SMILES string and molecule. Here the fact that multiple SMILES represent the same molecule is explored as a technique for data augmentation of a molecular QSAR dataset modeled by a long short term memory (LSTM) cell based neural network. The augmented dataset was 130 times bigger than the original. The network trained with the augmented dataset shows better performance on a test set when compared to a model built with only one canonical SMILES string per molecule. The correlation coefficient R2 on the test set was improved from 0.56 to 0.66 when using SMILES enumeration, and the root mean square error (RMS) likewise fell from 0.62 to 0.55. The technique also works in the prediction phase. By taking the average per molecule of the predictions for the enumerated SMILES a further improvement to a correlation coefficient of 0.68 and a RMS of 0.52 was found.

**Comment by mrwns** [comment]
also, this ICLR paper should be cited in the same context https://arxiv.org/abs/1602.06289

Learning to SMILE(S)

Stanisław Jastrzębski, Damian Leśniak, Wojciech Marian Czarnecki
(Submitted on 19 Feb 2016)
This paper shows how one can directly apply natural language processing (NLP) methods to classification problems in cheminformatics. Connection between these seemingly separate fields is shown by considering standard textual representation of compound, SMILES. The problem of activity prediction against a target protein is considered, which is a crucial part of computer aided drug design process. Conducted experiments show that this way one can not only outrank state of the art results of hand crafted representations but also gets direct structural insights into the way decisions are made.

**Comment by agitter** [comment]
Thanks, I added it as #326
